airavata_custos_portal/apps/frontend/urls.py

```diff
@@ -0,0 +1,28 @@
+from django.urls import path
+
+from . import views
+
+
+app_name = "airavata_custos_portal_frontend"
+urlpatterns = [
+    path('', views.home, name="home"),
```

Comment:
You can leave this if you want but you can use re_path to match multiple paths so you don't have to list them all out like this.